client/App.js

```diff
@@ -32,7 +33,14 @@ class App extends Component {
             fb.base.bindCollection(`Contributions`, {
                 context: this,
                 state: 'contributions',
-                withRefs: true
+                withRefs: true,
+                then: () => {
+                    fb.base.syncDoc("/Contributions/published", {
```

Comment:
These don't need to be done synchronously (aka you don't need to wait for bindContributions to finish to call syncDoc, they're independent and can happen async at the same time)